foundations/foundation-metrics/src/main/java/io/servicecomb/foundation/metrics/output/MetricsFileOutput.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2017 Huawei Technologies Co., Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.servicecomb.foundation.metrics.output;
+
+import java.util.Map;
+
+import com.netflix.config.DynamicPropertyFactory;
+
+public abstract class MetricsFileOutput {
+
+  public static final String METRICS_POLL_TIME = "servicecomb.metrics.polltime";
+  public static final String METRICS_FILE_ENABLED = "servicecomb.metrics.file.enabled";
+  public static final String METRICS_FILE_PATH = "servicecomb.metrics.file.file_root_path";
+  public static final String METRICS_FILE_SIZE = "servicecomb.metrics.file.max_rolling_size";
```

Comment:
The constant variable should be meaningful, Please add the important information about "rolling" to the constant.

Reply:
Fixed